tests/installation/scc_registration.pm

```diff
@@ -20,6 +20,16 @@ use testapi;
 use registration;
 
 sub run() {
+    if (get_var("SCC_EXPECT_ERROR")) {
+        while (1) {
+            if (check_screen 'registration-error') {
+                send_key 'alt-o';
+                sleep 2;
```

Comment:
I am still (slightly) human and I don't blindly wait two seconds and then press a key again ;-D wait_still_screen is exactly there to resemble human behaviour - this is how I also act - wait until something changes and then react. Sometimes I just assume the next screen to show the correct content and press the next key (`wait_still_screen`) without actually checking, which would be `assert_screen`. I am just saying "I don't like sleep's" and not everybody will agree so you can of course still decide on your own but my - so far limited - experience shows me that not relying on explicit sleeps makes better and more robust code and more readable what really the intention of the test writer was.